src/application-insights/azext_applicationinsights/_params.py

```diff
@@ -79,3 +79,48 @@ def load_arguments(self, _):
     with self.argument_context('monitor app-insights component linked-storage') as c:
         c.argument('storage_account_id', options_list=['--storage-account', '-s'], validator=validate_storage_account_name_or_id,
                    help='Name or ID of a linked storage account.')
+
+    with self.argument_context('monitor app-insights component continues-export list') as c:
+        c.argument('application', options_list=['--app', '-a'], id_part=None,
```

Comment:
It's redundant to register 'application' here as it's already registered in parent command.